embuilder.py

```diff
@@ -30,8 +30,8 @@
 
 # Minimal subset of targets used by CI systems to build enough to be useful
 MINIMAL_TASKS = [
-    'libcompiler_rt',
-    'libcompiler_rt-wasm-sjlj',
+    'libcompiler_rt-emsjlj',
```

Comment:
This suffix looks like it should no longer exist?

Reply:
Good catch, thanks. Fixed.